### ISSUE TITLE
Possible to run strategy in raspberry pi then execute trade in Windows 10?

### ISSUE THREAD

**Issue by cevangelio** [TheSnowGuru/PyTrader-python-mt4-mt5-trading-api-connector-drag-n-drop#13]
Hi, 

Is it possible to run the strategy from a raspberry pi then execute the trade order in Windows 10?

I tried to input the IP address of my Windows 10 computer and port used in the script being ran in the raspberry pi but I got error "BrokenPipeError: [Errno 32] Broken pipe".

Anything I can do to make this work?

**Comment by Branly76** [comment]
@cevangelio

I assume on the raspberry linux is running. I have not yet experience/knowledge with this combination. I have an EA bot running under a windows server (I use for remore testing). You can try this first. IP=85.215.82.243 and port number is 1122. MT4 EA is running.
Connected = MT.Connect(
    server='85.215.82.243',
    port=1122,
    instrument_lookup=brokerInstrumentsLookup).

It just happened i ordered 2x raspberry's to build a simple kubernetes cloud to run some python and also run MT4/MT5 under wine. Somewhere next week i will have the hardware and will start configuring and testing. So for this you need somepatience.

BR

**Comment by cevangelio** [comment]
Yes, the raspberry pi is running Raspbian and headless. No desktop, just CLI. The code you wrote is exactly what I did. I took the IP of my Windows 10 machine running the EA and port used and used that in the python script in the Raspberry pi. They are in the same network too. I did a ping from the raspberry pi to the Windows 10 computer and I do get a reply so the Windows is definitely reachable by the raspberry pi. If you can figure this out, that will be great. Will wait for your tests whne you got the raspberry pis.

**Comment by Branly76** [comment]
One of these days. Not sure if I have all hardware i need. But for sure it will be done.

BR

**Comment by cevangelio** [comment]
I just want to update you on this, I was able to figure out the issue. I think it is not Pytrader but my Windows 10. I am using Windows 10 home and it does not have remote connection enabled. Got to buy Pro edition (f**k MS!!!).

**Comment by Branly76** [comment]
Hi,

Thank you for the information.
I use windows 10 pro.

Have my hardware 3x PI4, and learning some linux to get kubernetes working.

BR
Huub

**Comment by Branly76** [comment]
Me again,

Look at this link. Windows 10 Pro for 15 euro.
I did buy twice there, site is in Dutch cann't help

https://www.123softwarelicentie.nl/shop/windows-10-pro/

BR
